### ISSUE TITLE
Allow themes to be in another folder

### ISSUE THREAD

**Issue by stefandoorn** [webgriffe/SyliusUpgradePlugin#5]
My `core-theme` is loaded using Composer and is therefore in a vendor folder. This core theme is again extended in a local theme in the `themes` folder, which is called `base-theme`.

For the `core-theme` I cannot do a diff:

```
bin/console webgriffe:upgrade:template-changes v1.7.11 v1.8.12 --theme=core-theme --legacy
ERROR: Cannot search "/var/www/html/symfony/themes/core-theme/" cause it does not exists.
```

The `base-theme` does work. 

Later I figured out this works, but I'm not sure this is intended to be working:

```
bin/console webgriffe:upgrade:template-changes v1.7.11 v1.8.12 --theme=../vendor/project/core-theme --legacy
```

**Comment by LucaGallinari** [comment]
Hi Stefan! Sorry for the late answer but i didn't receive the notification. Thank you for opening this issue, it gave me some ideas for possible features. However, that's a good workaround that i didn't thought of! I confirm that theoretically that was not intended but it must be possible to do that (without using a workaround). We have two possible options here:
* Let the user specify an absolute path in the "theme" option, pretty quick implementation but the option becomes a bit "confusing" (it allows both a relative path under the "themes" dir and an absolute path). And if you use theme inheritance you have to launch the command N times, but this could be mitigated by configuring the "theme" option to accept multiple values
* Add a new multiple option for absolute theme paths (that most likely will be the ones under vendor). But i don't know.. the difference between the two options (the actual for "local" paths and the new one for absolute paths) is really subtle.

Maybe the first one is ok, what do you think?

A possible extension to this implementation could be to handle automatically the inheritance of themes, but i'll open a different issue for that.

**Comment by stefandoorn** [comment]
Also no notifications here :-) I would be happy with a relative path (and a mention in the docs that due to that you can get to themes in the vendor folder also) + multiple `--theme` calls at once, that would be great :D

**Comment by LucaGallinari** [comment]
This has been included in release v0.5.0: https://github.com/webgriffe/SyliusUpgradePlugin/releases/tag/v0.5.0
